DA/server.ts

```diff
@@ -107,7 +108,21 @@ async function saveToDatabase(encTile: any) {
 
     socket.emit("saveToDatabaseResponse");
 }
-
+async function create_encrypted_tiles(){
```

Comment:
natural language paragraph comment above each function please. one or two sentences describing what + why is plenty